### ISSUE TITLE
Create pa11y dashboard for accessibility review

### ISSUE THREAD

**Issue by kenirwin** [alaoweb/2021.alaoweb.org#36]

**Comment by c-deems** [comment]
I was able to find Derek's instructions on creating a Pa11y Dashboard, and I'm going to add them here for the sake of sharing: https://github.com/alaoweb/pa11y-dashboard-dockerized

From my experience, it should just be a matter of plugging the URLs for the 2021 conference website into the dashboard, and it'll read the page and denote any errors that appear. I'll need to figure out how to set the dashboard up so that others have access to it though, as I've only ever set up dashboards for my own work. Once I do get it set up though, I'll be sure to post it here or send it out to everyone.

**Comment by dzoladz** [comment]
I'm willing to advise, if needed. Just ping me on Slack.

**Comment by c-deems** [comment]
The dashboard is now available for everyone to access, and can be done so by using the following URL: http://pa11y.alaoweb.org/

For the time being, I've populated the dashboard with just the main pages of the conference site (homepage, schedule, posters, etc) and ran each report under WCAG AA standards. I think this is a good baseline to work off of for now, especially since it seems that six errors that appear on each page's report are shared sitewide (color contrast of the text/links in the footer).

**Comment by kenirwin** [comment]
pally dashboard now linked in READEM page
